sql/7_0_2-to-7_0_3_upgrade.sql

```diff
@@ -203,3 +203,103 @@ ALTER TABLE `drugs` ADD `last_updated` DATETIME NOT NULL DEFAULT CURRENT_TIMESTA
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_developer', 'Developer of the intervention (translation from clinical research or guideline)', 20);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_funding', 'Funding source of the technical implementation for the intervention(s) development', 30);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_dates', 'Release and, if applicable, revision dates of the intervention or reference source', 40);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_race', 'Use of race as expressed in the standards in § 170.213', 50);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_ethnicity', 'Use of ethnicity as expressed in the standards in § 170.213', 60);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_language', 'Use of language as expressed in the standards in § 170.213', 70);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_sexual_orientation', 'Use of sexual orientation as expressed in the standards in § 170.213', 80);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_gender_identity', 'Use of gender identity as expressed in the standards in § 170.213', 90);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_sex', 'Use of sex as expressed in the standards in § 170.213', 100);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_dob', 'Use of date of birth as expressed in the standards in § 170.213', 110);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_sdoh', 'Use of social determinants of health data as expressed in the standards in § 170.213', 120);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('dsi_evidence_source_attributes', 'evidence_based_health_status', 'Use of health status assessments data as expressed in the standards in § 170.213', 130);
+#EndIf
+
+#IfNotTable dsi_source_attributes
+CREATE TABLE `dsi_source_attributes` (
+     `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+     `client_id` VARCHAR(80) NOT NULL,
+     `list_id` VARCHAR(100) NOT NULL,
+     `option_id` VARCHAR(100) NOT NULL,
+     `clinical_rule_id` VARCHAR(31) DEFAULT NULL,
+     `source_value` TEXT NOT NULL,
```

Comment:
for the TEXT, current practice is to not set DEFAULTS OR NULL to them (ie. just use TEXT and nothing else)

Reply:
Fixed